server/paymentProviders/thegivingblock/index.js

```diff
@@ -7,24 +7,55 @@ import fetch from 'node-fetch';
 import orderStatus from '../../constants/order_status';
 import { TransactionTypes } from '../../constants/transactions';
 import { getFxRate } from '../../lib/currency';
+import logger from '../../lib/logger';
 import { getHostFee, getHostFeeSharePercent } from '../../lib/payments';
 import models from '../../models';
 
 const AES_ENCRYPTION_KEY = config.thegivingblock.aesEncryptionKey;
 const AES_ENCRYPTION_IV = config.thegivingblock.aesEncryptionIv;
 const AES_ENCRYPTION_METHOD = config.thegivingblock.aesEncryptionMethod;
 const API_URL = config.thegivingblock.apiUrl;
-
-async function apiRequest(path, options = {}) {
+const GIVINGBLOCK_USERNAME = config.thegivingblock.username;
+const GIVINGBLOCK_PASSWORD = config.thegivingblock.password;
```

Comment:
```suggestion
const THEGIVINGBLOCK_PASSWORD = config.thegivingblock.password;
```

Consistency please, we never name anything "Giving Block" anywhere.

Reply:
Could actually be shorten to `PASSWORD` in this context, like other environment variables.

Reply:
Done. 👍🏽